### PR TITLE
feat: Created homepage and setup backend

### PR DESCRIPTION
**Pull request by jalajcodes**
Lots of files here as I am also creating backend along with frontend. (wish me luck lol😅)

Files to review:
```
frontend/src/App.js
frontend/src/Components/Footer.js
frontend/src/Components/Header.js
frontend/src/Components/Layout.js
frontend/src/Pages/Home.js
```

Live: https://max-store-app.herokuapp.com/

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
```
Comment (anik31) [inline frontend/src/App.js:1]: appreciation - saw a new way of browser router

File: frontend/src/Components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+              <a href="/">Compression</a>
+            </li>
+            <li>
+              <a href="/">Pricing</a>
+            </li>
+            <li>
+              <a href="/">FAQ</a>
+            </li>
+            <li>
+              <a href="/">Status</a>
```
Comment (anik31) [inline frontend/src/Components/Footer.js:23]: suggestion - I think `<a></a>` tag isn't required in react. You can use `<Link>` for navigating inside app & for external links you can use `<a>`

File: frontend/src/Components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      <span className="col">
+        <div className="subscribe-form-holder">
+          <h3 className="form-title">Subscribe to our email newsletter</h3>
+          <form action="#" className="subscribe-form">
+            <input
+              type="email"
+              name="email"
+              id="email"
+              className="email subscribe-input"
+              autoComplete="off"
```
Comment (anik31) [inline frontend/src/Components/Footer.js:72]: appreciation - good that you're using autocomplete

File: frontend/src/Components/Header.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            <input type="text" placeholder="Search" />
+            <button>
+              <i className="fas fa-search" />
+            </button>
+          </div>
+          <a href="./product.html" className="nav__link">
+            <i className="fa-solid fa-crown" />
+            <span className="nav__link--text">Products</span>
+          </a>
+          <a href="./wishlist.html" className="nav__link">
```
Comment (anik31) [inline frontend/src/Components/Header.js:23]: suggestion - replace `<a>` with `<Link>`

File: frontend/src/Pages/Home.js [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+const Home = () => {
+  return (
+    <section className="home-page">
+      <div className="cta container">
+        <div className="cta-text">
+          <h1>
+            All You Need for Your <span>Wardrobe</span>
+          </h1>
+          <p>
+            Shop now to get <img src="./images/max.png" alt="" /> discount.
```
Comment (anik31) [inline frontend/src/Pages/Home.js:12]: suggestion - write something about the contents of image in alt

**Review by anik31 — COMMENTED** [review]
Good work 👍 Code is clean & UI is also good

File: frontend/src/Components/Header.js (hunk 4 above)
Comment (jalaj-gl) [reply frontend/src/Components/Header.js:23]: Yes I am aware of these will do it soon.

File: frontend/src/Pages/Home.js (hunk 5 above)
Comment (jalaj-gl) [reply frontend/src/Pages/Home.js:12]: Will do that. Thanks!

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me. The website also looks great!

Just make the suggested changes in future PRs

**Pull request merged**